mnist/mlcube.yaml

```diff
@@ -1,11 +1,24 @@
-schema_version: 1.0.0
-schema_type: mlcube_root
-
 name: mnist
-author: MLPerf Best Practices Working Group
-version: 0.1.0
-mlcube_spec_version: 0.1.0
+description: MLCommons MNIST MLCube example
+authors:
+  - {name: "Sergey Serebryakov", email: "sergey.serebryakov@hpe.com", org: "Hewlett-Packard Labs"}
+
+platform:
+  accelerator_count: 0
+
+container:
+  image: mlcommons/mnist:0.0.1        # Image name.
+  build_context: "mnist"              # Docker build context relative to $MLCUBE_ROOT. Default is `build`.
+  build_file: "Dockerfile"            # Docker file name within docker build context, default is `Dockerfile`.
 
 tasks:
-  - 'tasks/download.yaml'
-  - 'tasks/train.yaml'
+  download:
+    io:
+      - {name: data_dir, type: directory, io: output, default: $WORKSPACE/data}
```

Comment:
Let's remove "$WORKSPACE/" as prefix since it's always there; so we just say it's implied.

Instead it is just "default: data"